pineappl_py/pineappl/channel.py

```diff
@@ -0,0 +1,16 @@
+from .pineappl import PyChannel
```

Comment:
We shouldn't explicitly create Python modules anymore and let PyO3 do that.

Reply:
Yes, indeed (cc https://github.com/NNPDF/pineappl/issues/200). I am planning to do this soon.

Reply:
addressed in #302